app/src/main/java/com/sun/mangareader01/ui/search/SearchPresenter.kt

```diff
@@ -0,0 +1,24 @@
+package com.sun.mangareader01.ui.search
+
+import com.sun.mangareader01.data.model.MangaSearchResponse
+import com.sun.mangareader01.data.source.local.OnLoadedDataCallback
+import com.sun.mangareader01.data.source.repository.MangaRepository
+
+class SearchPresenter(
+    private val searchView: SearchFragment,
```

Comment:
```suggestion
    private val view: SearchContract.View,
```
em nên khai báo là interface contract
như vậy hàm contract e define mới có giá trị